--- gtk-vnc-0.4.4/src/Makefile.am.orig	2011-11-13 14:03:14.231161724 +0100
+++ gtk-vnc-0.4.4/src/Makefile.am	2011-11-13 14:30:58.397884111 +0100
@@ -318,6 +318,7 @@
 		--include GObject-2.0 \
 		$(PREFIX_ARGS) \
                 --library=$(builddir)/libgvnc-1.0.la \
+		--libtool="$(LIBTOOL)" \
                 --add-include-path=$(top_srcdir) \
                 --add-include-path=$(srcdir) \
                 --add-include-path=$(builddir) \
@@ -339,6 +339,7 @@
 		--include GVnc-1.0 \
 		$(PREFIX_ARGS) \
                 --library=$(builddir)/libgtk-vnc-$(GTK_VNC_API_VERSION).la \
+		--libtool="$(LIBTOOL)" \
                 --add-include-path=$(top_srcdir) \
                 --add-include-path=$(srcdir) \
                 --add-include-path=$(builddir) \
